pkg/kubelet/kubelet.go

```diff
@@ -1152,120 +1152,183 @@ func (kl *Kubelet) pullImageAndRunContainer(pod *api.BoundPod, container *api.Co
 	return containerID, nil
 }
 
-func (kl *Kubelet) syncPod(pod *api.BoundPod, containersInPod dockertools.DockerContainers) error {
+type podContainerChangesSpec struct {
```

Comment:
Done. I guess this is a good place to describe semantics, so it's a long comment.